### ISSUE TITLE
Feature Request: multi decks

### ISSUE THREAD

**Issue by iamdeveloperidevelop** [Kamalisk/arkhamdb#89]
Something similar to http://ringsdb.com/ 
This feature is almost must have, I do not see sense in "multiplayer" while can not connect more than one deck with each other.

**Comment by mattsigal** [comment]
Not sure this is exactly what you are looking for, as I've never used ringsdb.com, but what I've been doing to indicate decks built together is to use the tags function.  For example:

![screen shot 2017-05-16 at 12 12 39 pm](https://cloud.githubusercontent.com/assets/1421758/26116419/133757be-3a31-11e7-86ef-d4013b6cf9e3.png)

All you need to do is select all the decks that are in the same campaign, then you can click "With Selection -> Add One or More Tags" then make a tag for that campaign. Then when browsing your decks, you can quickly switch between campaigns:

![screen shot 2017-05-16 at 12 14 54 pm](https://cloud.githubusercontent.com/assets/1421758/26116484/5012275e-3a31-11e7-811b-6ad397aed260.png)

**Comment by iamdeveloperidevelop** [comment]
Thanks for reply. That is much help. Still, gonna show you screen from ringsdb.com just to your knowledge if you do not mind. I do think one merging view would be helpful, and not much effort (if have time, cause I have no knowledge about php and cant help :( ). So you see all cards from all tagged builds, without need to jump between cards and make notes for tags:

![ringsdb](https://cloud.githubusercontent.com/assets/3028959/26117732/bed02f8e-3a67-11e7-9fd3-0f2fa6e8d349.png)

The dream would be like merged statistics for all characters in tag/group. Food for thought ;)
Still, thanks for tag tip :)

**Comment by Kamalisk** [comment]
I would like to have this eventually, especially for publishing deck lists meant to work together. Not as simple as copying ringsdb though as they have added a lot of stuff (I based it on netrunnerdb when I started)

**Comment by sacrelicious2** [comment]
I'm not sure if ringsdb.com does this, but it would be nice to be able to specify that the decks share a given collection of packs, so you know you are limited on certain cards between decks (So not everyone can run 2x Fine Clothes in a 4-player game for example).  If this is too difficult, then it would be nice if we could see how many copies of cores + packs the set of decks requires.

**Comment by chrisheart86** [comment]
Yeah, I'm just editing decks for a 2 player game and was looking for something like linking decks so I can see how many of the shared cards are available.
EDIT: Also, I just realised, the exact amount of cards in the sets owned by me are not accounted for here, I just added two copies of .41 Derringer to my Wendy deck, only realising that there's only 1 in core, of which I only own 1. It'd be nice to keep track of this somehow.
